Formula/clang-format.rb

```diff
@@ -25,15 +16,8 @@ class ClangFormat < Formula
   end
 
   head do
-    url "https://git.llvm.org/git/llvm.git"
-
-    resource "clang" do
-      url "https://git.llvm.org/git/clang.git"
-    end
-
-    resource "libcxx" do
-      url "https://git.llvm.org/git/libcxx.git"
-    end
+    depends_on "git" => :build
```

Comment:
Where is this dependency used?

Reply:
not sure where. there was svn, switched it to git. You think it's useless?

Reply:
Can you try without it? That'll make it clear if it's useless

Reply:
it uses git to checkout from git. So, should be present

Reply:
It does checkouts beyond the one defined on line 8?

Reply:
Xcode comes with git. Homebrew heavily relies on it, and there's in fact a shim in place for it which is used in formula builds. Unless there is a specific need to install a newer git, the `depends_on` is not needed.

Reply:
@Bo98 ok, will remove